opencart 1.5.6.4/upload/catalog/view/theme/default/template/module/smaily_for_opencart.tpl

```diff
@@ -0,0 +1,35 @@
+<div class="panel">
+  <div class="container-fluid">
+      <h3>Subscribe to newsletter</h3>
```

Comment:
Shouldn't the title be translatable?

Reply:
You have made it translatable, but in the process you have broken the formatting (indentation).